.github/workflows/release.yml

```
@@ -0,0 +1,31 @@
+# release process. To enable this workflow again, restore the `on` block below
+# and remove the `if: false` condition in the job.
+#
+# on:
+#   push:
+#     branches:
+#       - main
+#     paths:
+#       - 'app/**'
+#   workflow_dispatch:
+
+jobs:
```

actionlint (1.7.7)

"on" section is missing in workflow

(syntax-check)